docs/install.md

```diff
@@ -152,6 +152,9 @@ On Debian or Ubuntu, you should be able to get all these with:
         golang-go
     curl https://install.meteor.com/ | sh
 
+You may need to do some extra work to get Clang 6, Debian and
+Ubuntu derivatives can use https://apt.llvm.org/
```

Comment:
It might be better to (a) put this before the commands above; users are less likely to install the old version and only then find out that they need a different one, and (b) mention which versions of ubuntu/debian have a new enough version, and which need to use the llvm project's repos.

Reply:
Good points.  I'll do a little research and resubmit.